src/Tribe/Settings.php

```diff
@@ -607,13 +607,13 @@ public function deleteOptions() {
 		public function get_url( array $args = array() ) {
 			$defaults = array(
 				'page' => $this->adminSlug,
+				'parent' => self::$parent_page,
 			);
 
 			// Allow the link to be "changed" on the fly
 			$args = wp_parse_args( $args, $defaults );
-			$parent = self::$parent_page;
 
-			$url = admin_url( $parent );
+			$url = admin_url( $args['parent'] );
```

Comment:
Great point on 1. I fell relatively confident on the 2nd point and have tested Community Tickets as well as noted that it should be QA'd in the ticket.